cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -387,6 +388,16 @@ def _translate_case(self):
             request_data.update(translate_employment(self.case_data.you.income, self.case_data.you.deductions))
         return request_data
 
+    def _translate_response(self, cfe_response):
```

Comment:
I had expected this would be a method in CfeResponse. WDYT?

Reply:
I guess it's preference. I think OO theory is divided on this subject e.g. https://stackoverflow.com/questions/57009803/object-oriented-design-pattern-for-object-converter

It's just personally I think of a translator to a different type as being a separate thing. CfeResponse is an encapsulation of the response from CFE, and I wouldn't expect to find one of the translators in it.

tbh I'm not a fan of CfeResponse either - rather formal OO for my taste. But can see its methods might be useful.

The tuple response is quite Pythonic, and I guess there's the zen about being explicit - it's really clear this way what it returns, rather than having to dig into the encapsulation to understand that the translation of the response is available.

Reply:
I guess if we're returning True, False, 'unknown' then it sits better here where it can be seen by the consumer.